constellations/orion/src/domains/class.go

```diff
@@ -25,6 +25,7 @@ type Class struct {
 	GoogleClassCode NullString `json:"googleClassCode" db:"google_class_code"`
 	FullState       int        `json:"fullState" db:"full_state"`
 	PricePerSession NullUint   `json:"pricePerSession" db:"price_per_session"`
+	PriceLump       NullUint   `json:"priceLump" db:"price_lump"`
 }
 
 // Class Methods
```

Comment:
Can you add a validation + validationTest?

Classes should either have a PricePerSession or PriceLump.
So in other words, a valid class should have a non-null, non-zero PricePerSession OR a non-null PriceLump

Reply:
can there be both?

Reply:
for now, no.

Reply:
so either or then